tests/integration/targets/vmware_folder_template_from_vm/tasks/main.yml

```diff
@@ -98,7 +142,21 @@
         cluster: "{{ vcenter_cluster_name }}"
         port: "{{ vcenter_port }}"
         datacenter: "{{ vcenter_datacenter }}"
-        folder: "{{ template_folder }}"
+        folder: "{{ item }}"
         state: absent
         force: true
         name: "{{ template_name }}"
+      loop:
+        - "{{ vm_folder }}"
+        - "{{ template_folder }}"
+
+    - name: "Test teardown: Delete Folder {{ template_folder }}"
```

Comment:
what about `vm_folder`? Should it be deleted too?

Reply:
`vm_folder` is our `e2e-qe` folder which already exists and other tests are relying on its existence, so we don't delete it

Reply:
We might want to create the VM in a disposable folder instead. WDYT?

Reply:
I think it's better create a new folder

Reply:
Done. 
BTW if we don't want to depend on the pre-existence of the `e2e-qe` VM folder then we should use the same approach in vmware_content_template integration tests [here](https://github.com/ansible-collections/vmware.vmware/blob/main/tests/integration/targets/vmware_content_template/defaults/main.yml#L6) and in the validated content tests as well

Reply:
yes, you are right